waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -916,7 +916,29 @@
            :resources {:limits {:memory memory}
                        :requests {:cpu cpu :memory memory}}
            :volumeMounts [{:mountPath "/srv/www"
-                           :name "user-home"}]})))))
+                           :name "user-home"}]}))
+
+      ;; Optional envoy sidecar container
+      (pos-int? offset)
```

Comment:
This should probably be predicated on `has-reverse-proxy?` rather than a positive port offset.

Reply:
Done